api/openapi.yaml

```diff
@@ -1100,11 +1100,11 @@ paths:
         content:
           text/json:
             schema:
-              $ref: "./config.yaml#/components/schemas/TestConnectionRequest"
+              $ref: "./config.yaml#/components/schemas/DataStoreTestConnectionRequest"
```

Comment:
Should we change the path `/config/connection` to something more data store specific? And should we document the `4xx` status code when the test connection fails?